samples/snippets/core/system-text-json/csharp/Utf8ReaderPartialRead.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Text.Json;
+
+namespace SystemTextJsonSamples
+{
+    public class Utf8ReaderPartialRead
+    {
+        public static void Run()
+        {
+            var jsonString = @"{
+                ""Date"": ""2019-08-01T00:00:00-07:00"",
+                ""Temperature"": 25,
+                ""TemperatureRanges"": {
+                    ""Cold"": { ""High"": 20, ""Low"": -10 },
+                    ""Hot"": { ""High"": 60, ""Low"": 20 }
+                },
+                ""Summary"": ""Hot"",
+            }";
+
+            var bytes = System.Text.Encoding.UTF8.GetBytes(jsonString.Trim());
+            var stream = new MemoryStream(bytes);
+
+            var buffer = new byte[10];
+            var span = new Span<byte>(buffer);
+
+            // Fill the buffer
+            var read = stream.Read(span);
+
+            var reader = new Utf8JsonReader(span, false, state: default);
+            Console.WriteLine($"String in buffer is: {System.Text.Encoding.UTF8.GetString(span)}");
+
+            // Search for "Summary" property name
+            while (reader.TokenType != JsonTokenType.PropertyName || reader.GetString() != "Summary")
```

Comment:
Use ValueTextEquals when comparing and searching for a property name. It is more efficient because you don't have to create/allocate a string for all the properties that you are not looking for along the way.
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.valuetextequals?view=netcore-3.1#System_Text_Json_Utf8JsonReader_ValueTextEquals_System_String_

```suggestion
            while (reader.TokenType != JsonTokenType.PropertyName || !reader.ValueTextEquals("Summary"))
```